code/modules/spells/aoe_turf/conjure/forcewall.dm

```diff
@@ -41,7 +41,10 @@
 		for(var/mob/M in T)
 			Proj.on_hit(M,M.bullet_act(Proj, def_zone))
 	return
-
+	
+/obj/effect/forcefield/wizard
+	alpha = 255
```

Comment:
I have no idea
I assumed it was something to do with being visible when mapping?